taker-frontend/src/useEventSource.ts

```diff
@@ -1,9 +1,7 @@
-import { useToast } from "@chakra-ui/react";
-import { useEffect, useRef, useState } from "react";
+import { useEffect, useState } from "react";
 
-export function useEventSource(url: string, withCredentials?: boolean) {
+export function useEventSource(url: string, withCredentials?: boolean, setIsConnected) {
```

Comment:
I think we could easily remove the `withCredentials` argument and always set it to true inside here. We never need this _without_ that flag set.

Reply:
we never use a different url either.

Reply:
Fair point. Feels weird though to hardcode the URL.